src/script-runner.js

```diff
@@ -34,26 +91,47 @@ window.web3Provider = {
 }
 window.web3 = new Web3(window.web3Provider)
 
-window.starknet = starknet
-
-console.log = function () {
-  window.remix.emit('log', {
-    data: Array.from(arguments)
+// 'getContractFactory' is added to 'ethers' to support usage of hardhat.ethers in the tests
+window.ethers.getContractFactory = (contractName, options) => {
```

Comment:
please make sure it fits with the current interface:
e.g the 2nd optional parameter is `signer?: ethers.Signer` , not `options`
the list os interfaced is there https://hardhat.org/plugins/nomiclabs-hardhat-ethers.html#helpers

Reply:
This is implemented.